config/sources/families/media.conf

```diff
@@ -17,7 +17,7 @@ elif [[ $BOARD == station-m3 ]]; then
 	BOOTBRANCH='branch:rk3588'
 	BOOTPATCHDIR="u-boot-station-p2"
 else
-	BOOTBRANCH="tag:v2022.07"
+	BOOTBRANCH="tag:v2022.04"
```

Comment:
I see that many people write about a large number of bugs in the work of 22.07 (for me, the main thing is that the USB keyboard does not work and you cannot make a system selection when starting in the EFI\extlinux.conf menu). Therefore, I rolled back to version 22.04, it is stable and works well. As far as possible, I plan to test 22.10\rcN, if there are no problems there, it will be possible to upgrade to the new version.

Reply:
OK. Forget about .10 ... its in worse state.

Reply:
> Forget about .10 ... its in worse state.

Ok  :)

By the way, the u-boot Helios64 assembly works correctly in 22.04

Reply:
I know that. We have to draw the line here. HW needs a maintainer and that can't be people that doesn't have HW nor time dealing with.

Reply:
Ok